nj2k/src/org/jetbrains/kotlin/nj2k/conversions/DefaultArgumentsConversion.kt

```diff
@@ -102,7 +105,12 @@ class DefaultArgumentsConversion(context: NewJ2kConverterContext) : RecursiveApp
                             return JKFieldAccessExpression(newSymbol)
                         }
                     }
-
+                    if (on is JKExpression) {
```

Comment:
Please, let's don't break functionallity with remapping we already have. Join your `addThisReceiverIfNeeded` with `remapParameterSymbol`, so we will be able to do both and not just taking over one case and forgetting about other one